src/psyclone/tests/psyGen_test.py

```diff
@@ -2386,3 +2389,299 @@ def test_dataaccess_same_vector_indices(monkeypatch):
+    literal = Literal("1")
+    coloredtext = colored("Literal", SCHEDULE_COLOUR_MAP["Literal"])
+    literal.view()
+    output, _ = capsys.readouterr()
+    assert coloredtext+"[value:'1']" in output
+
+
+def test_literal_can_be_printed():
+    '''Test that an Literal instance can always be printed (i.e. is
+    initialised fully)'''
+    literal = Literal("1")
+    assert "Literal[value:'1']\n" in literal.__str__()
+
+
+# Test BinaryOperation class
+
+
+def test_BinaryOperation_view(capsys):
+    ''' Check the view and colored_text methods of the Binary Operation
+    class.'''
+    from psyclone.psyGen import colored, SCHEDULE_COLOUR_MAP
+    binaryOp = BinaryOperation("+")
+    op1 = Literal("1", parent=binaryOp)
+    op2 = Literal("1", parent=binaryOp)
+    binaryOp.addchild(op1)
+    binaryOp.addchild(op2)
+    coloredtext = colored("BinaryOperation",
+                          SCHEDULE_COLOUR_MAP["BinaryOperation"])
+    binaryOp.view()
+    output, _ = capsys.readouterr()
+    assert coloredtext+"[operator:'+']" in output
+
+
+def test_BinaryOperation_can_be_printed():
+    '''Test that a Binary Operation instance can always be printed (i.e. is
+    initialised fully)'''
+    binaryOp = BinaryOperation("+")
+    op1 = Literal("1", parent=binaryOp)
+    op2 = Literal("1", parent=binaryOp)
+    binaryOp.addchild(op1)
+    binaryOp.addchild(op2)
+    assert "BinaryOperation[operator:'+']\n" in binaryOp.__str__()
+
+
+# Test fparser2ASTProcessor
+
+
+def test_fparser2ASTProcessor_handling_assignments():
+    from fparser.two.parser import ParserFactory
+    from fparser.common.readfortran import FortranStringReader
+    from fparser.two.Fortran2003 import Execution_Part
+    ParserFactory().create(std="f2008")
+    reader = FortranStringReader("x=1")
+    fparser2assignment = Execution_Part.match(reader)[0][0]
+
+    fake_parent = Node()
+    processor = fparser2ASTProcessor()
+    processor.process_nodes(fake_parent, [fparser2assignment], None)
+    # Check a new node was generated a connected to parent
+    assert len(fake_parent.children) == 1
+    new_node = fake_parent.children[0]
+    assert isinstance(new_node, Assignment)
+    assert len(new_node.children) == 2
+
+
+def test_fparser2ASTProcessor_handling_Names():
+    from fparser.two.parser import ParserFactory
+    from fparser.common.readfortran import FortranStringReader
+    from fparser.two.Fortran2003 import Execution_Part
+    ParserFactory().create(std="f2008")
+    reader = FortranStringReader("x=1")
+    fparser2name = Execution_Part.match(reader)[0][0].items[0]
+
+    fake_parent = Node()
+    processor = fparser2ASTProcessor()
+    processor.process_nodes(fake_parent, [fparser2name], None)
+    # Check a new node was generated a connected to parent
+    assert len(fake_parent.children) == 1
+    new_node = fake_parent.children[0]
+    assert isinstance(new_node, Reference)
+    assert new_node._reference == "x"
+
+
+def test_fparser2ASTProcessor_handling_Parenthesis():
```

Comment:
lower case Parenthesis? Please be consistent with test names and check other tests.